src/main/java/dk/cngroup/lentils/service/UserService.java

```diff
@@ -0,0 +1,34 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.User;
+import dk.cngroup.lentils.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class UserService {
+
+    public static final Long ORGANIZER_ID = 2L;
```

Comment:
Tak tady bych nedával IDčko. Spíš bych hledal podle toho jména role, string "organizer".

ID se může stát, že se ti to změní (update db, dump recovery), jméno té role by ale mělo být pořád stejné.